posthog/urls.py

```diff
@@ -29,125 +24,15 @@
 )
 from posthog.demo import demo
 from posthog.email import is_email_available
-from posthog.event_usage import report_user_signed_up
 
-from .api.signup import SignupSerializer
-from .models import OrganizationInvite, Team, User
-from .utils import get_can_create_org, render_template
+from .utils import render_template
 from .views import health, login_required, preflight_check, robots_txt, stats
 
 
 def home(request, *args, **kwargs):
     return render_template("index.html", request)
 
 
-class TeamInviteSurrogate:
```

Comment:
Signup-related methods & classes are moved to `signup.py` file. Imports are moved accordingly. Where there's a change in code for these methods/classes, it's marked in the new file.